### PR TITLE
Task 5: Import to S3

### PR DESCRIPTION
**Pull request by DanilaFadeev**
### Description:
- [X] File `serverless.yml` contains configuration for `importProductsFile` function **(1 point)**
- [X] The `importProductsFile` lambda function returns a correct response which can be used to upload a file into the S3 bucket **(3 points)**
- [X] Frontend application is integrated with `importProductsFile` lambda **(4 points)**
  - https://d40ridgoulz65.cloudfront.net/admin/products
  - https://github.com/DanilaFadeev/shop-react-redux-cloudfront/pull/4
- [X] The `importFileParser` lambda function is implemented and serverless.yml contains configuration for the lambda **(5 points)**

### Additional (optional) tasks
- [X] async/await is used in lambda functions **(1 point)**
- [X] `importProductsFile` lambda is covered by unit tests **(1 point)**
- [X] At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder) **(1 point)**

Total points: 8/8

### Screenshots:
Screenshot 2021-09-23 at 01 47 08
Screenshot 2021-09-23 at 01 49 00
Screenshot 2021-09-23 at 01 49 20

### CODE REVIEW

File: src/import-service/src/functions/import-file-parser/handler.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          const deleteObjectCommand = new DeleteObjectCommand({
+            Bucket: s3Bucket,
+            Key: records[0].s3.object.key
+          });
+          const response22 = await s3Client.send(deleteObjectCommand);
+          console.log(response22);
+
+          resolve(void 0);
+        })
+        .on('err', reject);
```
Comment (belspirit) [inline src/import-service/src/functions/import-file-parser/handler.ts:61]: Will be better to log the `error.message`

Comment (DanilaFadeev) [reply src/import-service/src/functions/import-file-parser/handler.ts:61]: The whole promise will be rejected which will cause an error. Function execution will be terminated and error will be printed into logs 🙂

Comment (belspirit) [reply src/import-service/src/functions/import-file-parser/handler.ts:61]: You are right, I missed rejected

Comment (belspirit) [reply src/import-service/src/functions/import-file-parser/handler.ts:61]: From a testing perspective, how would you know what is the issue?

**Pull request merged**